.github/workflows/instrument.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   build:
     strategy:
       matrix:
-        haxe-version: ['4.1.5', '4.2.1', 'nightly']
+        haxe-version: ['4.1.5', '4.2.1', '4.2.3', 'nightly']
```

Comment:
I think we can drop 4.2.1 entirely

Reply:
Done. You can choose to squash the commits if you wish.